ml/README.md

```diff
@@ -0,0 +1,249 @@
+
+```
+        dimensions
+time    d1	d2	d3	d4	d5		NAR
+   1	0	0	0	0	0		 0%
+   2	0	0	0	0	100		20%
+   3	0	0	0	0	0		 0%
+   4	0	100	0	0	0		20%
+   5	100	0	0	0	0		20%
+   6	0	100	100	0	100		60%
+   7	0	100	0	100	0		40%
+   8	0	0	0	0	100		20%
+   9	0	0	100	100	0		40%
+   10	0	0	0	0	0		 0%
+ 
+DAR	    10%	30%	20%	20%	30%		22% NAR_t1-t10
+
+DAR        = Dimension Anomaly Rate
+NAR        = Node Anomaly Rate
+NAR_t1-t10 = Node Anomaly Rate over t1 to t10
+```
+
+Here we can just average a row or a column and work out an ["anomaly rate"](#anomaly-rate) in any direction. For example, if we were to just average along a row then this would be the ["node anomaly rate"](#node-anomaly-rate) (all dimensions) at time t. Likewise if we averaged a column then we would have the ["dimension anomaly rate"](#dimension-anomaly-rate) for each dimension over the time window t=1-10. Extending this idea we can work out an overall ["anomaly rate"](#anomaly-rate) for the whole matrix or any subset of it we might be interested in.
+
+### Anomaly Detector - Node level anomaly events
+
+Netdata uses the concept of an ["anomaly detector"](#anomaly-detector) that looks at all the anomaly bits of the node and if a sufficient percentage of them are high enough for a persistent amount of time then an ["anomaly event"](#anomaly-event) will be produced. This anomaly event signals that there was sufficient evidence among all the anomaly bits that some strange behavior might have been detected in a more global sense across the node.
+
+Essentially if the ["Node Anomaly Rate"](#node-anomaly-rate) (NAR) passes a defined threshold and stays above that threshold for a persistent amount of time a "Node [Anomaly Event](#anomaly-event)" will be triggered.
+
+These anomaly events are currently exposed via `/api/v1/anomaly_events`
+
+https://london.my-netdata.io/api/v1/anomaly_events?after=1638365182000&before=1638365602000
+
+If an event exists within the window the result would be a list of start and end time's
+
+```
+[
+    [
+        1638367788,
+        1638367851
+    ]
+]
+```
+
+Where information about each anomaly event can then be found at the `/api/v1/anomaly_event_info` endpoint (making sure to pass the `after` and `before` params): 
+
+https://london.my-netdata.io/api/v1/anomaly_event_info?after=1638367788&before=1638367851
+
+```
+[
+    [
+        0.66,
+        "netdata.response_time|max"
+    ],
+    [
+        0.63,
+        "netdata.response_time|average"
+    ],
+    [
+        0.54,
+        "netdata.requests|requests"
+    ],
+    ...
+```
+
+This returns a list of dimension anomaly rates for all dimensions that were considered part of the detected anomaly event.
+
+**Note**: We plan to build additional anomaly detection and exploration features into both Netdata Agent and Netdata Cloud and so these endpoints are primarily to power these features and so still under active development.
+
+## Configuration
```

Comment:
Maybe you have more info on the typical location of the config file?

Reply:
yep makese sense - added a link to here: https://learn.netdata.cloud/guides/step-by-step/step-04